src/pcb/pcb_smtpad.ts

```diff
@@ -53,6 +55,7 @@ export interface PcbSmtPadCircle {
   port_hints?: string[]
   pcb_component_id?: string
   pcb_port_id?: string
+  no_solder_paste?: boolean
```

Comment:
I'm not sure i understand what this is from the name, i think it means "should_not_infer_solder_paste" but i think it can probably be removed (solder paste can completely be taken care of by the other element)